rest/src/main/java/org/jboss/pnc/rest/provider/BuildConfigurationProvider.java

```diff
@@ -173,4 +175,15 @@ public void removeProductVersion(Integer configId, Integer productVersionId) {
         buildConfigurationRepository.save(buildConfig);
     }
 
+    public List<ProductVersionRest> getProductVersions(Integer configId) {
+        BuildConfiguration buildConfig = buildConfigurationRepository.findOne(configId);
```

Comment:
If we do it this way, we won't be able to use DB side logic, which is pretty bad. I would prefer if you used a proper repository for this and not a getter.